Xamarin.Forms.Platform.Android/Renderers/EntryRenderer.cs

```diff
@@ -109,6 +109,8 @@ protected override void OnElementChanged(ElementChangedEventArgs<Entry> e)
 			UpdateMaxLength();
 			UpdateImeOptions();
 			UpdateReturnType();
+			UpdateCursorSelection();
```

Comment:
@almirvuk it looks like the addition of this call here is causing this UI Test to fail
```
Xamarin.Forms.Controls.Issues.Bugzilla46458.GridIsEnabled
```

If you don't have time to look at this can you give me permissions to push to this branch? Then I can just try to resolve?

Reply:
@almirvuk I realize this was 5 months ago and I know I wouldn't remember :-) but do you remember why you added an update call here to *UpdateCursorSelection*  ? 

https://github.com/xamarin/Xamarin.Forms/commit/8a0efac709b83343fdbe0eb4e872ffdd06f6364b